dev/macos/setup.sh

```diff
@@ -11,8 +11,6 @@ sudo cp minio /usr/local/bin/minio
 
 # restoring the default for changeps1 to have parity with dev
 conda config --set changeps1 true
-# make sure the caching works correctly
-conda config --set use_only_tar_bz2 true
```

Comment:
Why?

Reply:
See https://github.com/conda/conda/issues/12332 and https://github.com/conda-incubator/setup-miniconda/issues/267.

Newer packages on conda-forge are only using .conda artifacts, so latest conda-build won't be found without turning this off.